polysemy-plugin/src/Polysemy/Plugin.hs

```diff
@@ -93,10 +93,11 @@ installTodos todos = do
   case optLevel dflags of
     0 -> pure todos
     _ -> do
-      mods <- moduleSetElts <$> getVisibleOrphanMods
-      pure $ todos
 #if __GLASGOW_HASKELL__ >= 810
-          ++ bool []
-                  (extraPhases dflags)
-                  (any ((== polysemyInternal) . moduleName) mods)
+      mods <- moduleSetElts <$> getVisibleOrphanMods
```

Comment:
Just to ease my mind, did you check this branch to make sure it compiles?

Reply:
Yes - is there something that seems questionable?

Reply:
Nope, nothing! Just being overly cautious!